tests/php/ORM/DBSchemaManagerTest/ChildClass.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace SilverStripe\ORM\Tests\DBchemaManagerTest;
+
+class ChildClass extends BaseClass
+{
+    private static $table_name = 'DBchemaManagerTest_ChildClass';
```

Comment:
```suggestion
    private static $table_name = 'DBSchemaManagerTest_ChildClass';
```

Reply:
and within all other test classes as well

Reply:
Fixed